caluma/form/tests/test_form.py

```diff
@@ -0,0 +1,23 @@
+from ...schema import schema
+
+
+def test_all_forms_connection(db, form, snapshot):
+    query = """
+    query AllFormsQuery {
+      allForms {
+        edges {
+          node {
+            id
+            slug
+            name
+            description
+            meta
+          }
+        }
+      }
+    }
+    """
+
+    result = schema.execute(query)
+    assert not result.errors
+    snapshot.assert_match(result.data)
```

Comment:
Maybe add a test for fetching a single form (node graph) aswell?

Reply:
Added a generic global node test.